app/Http/Controllers/Setup/Controller.php

```diff
@@ -24,20 +25,26 @@ public function setup() {
           || !is_writable(__DIR__.'/../../../../storage/logs/.gitignore')) {
             return view('setup/permissions');
         }
-
+        if(!empty(env('DATABASE_URL'))) {
```

Comment:
I want the `setup/start` to be the first thing people see unless they have file permissions issue. I think the intent of this is to show the database config on heroku if it's not set in the environment, so that should be accomplished some other way instead.

Reply:
The intent is likely to only show database page if the database url is not set in the environment

Reply:
I can remove a redirect 👍